core/indigo-core/common/math/algebra.h

```diff
@@ -303,6 +303,54 @@ namespace indigo
             _rightTop = other._rightTop;
         }
 
+        inline bool pt_in_rect(const Vec2f& pt) const
```

Comment:
Please avoid abbreviations in method names.
Looking at other names, here was used camelCase style, not Snake case. Let's follow the style of the header.

Reply:
fixed